### ISSUE TITLE
[Question] 0/1 nodes are available

### ISSUE THREAD

**Issue by Eminos2** [harvester/harvester#3025]
Hi Guys, i am very new to Harvester. 
I installed it on a Server and now i am trying to set up my first virtual Machine. But i keep getting that no nodes are available.
![image](https://user-images.githubusercontent.com/72886423/197493546-dacdb21f-bdd7-4d04-89af-b8581a2667d1.png)
![image](https://user-images.githubusercontent.com/72886423/197493727-6071b065-3333-4ab9-9810-66c7a999f5a7.png)

How can i configure the nodes and could you help me to figure out, what is the exact problem. Thank you!

With regards
Eminos2

**Comment by w13915984028** [comment]
How is your Harvester cluster insalled, via ISO or `dev-mode` ? 

And what is your VM's specification, cores, memeory, storage.

**Comment by Eminos2** [comment]
Hi, it is installed via ISO.

VM: 2 Cores, 4 Gi Memory, 10 Gi Storage.

**Comment by w13915984028** [comment]
Please add a support bundle, thanks.

https://docs.harvesterhci.io/v1.0/troubleshooting/harvester/#generate-a-support-bundle

**Comment by w13915984028** [comment]
And also a few pictures about the detail of the VM in Harvester WebUI, thanks.

**Comment by Eminos2** [comment]
Hi thank you, 
i tried several variations to create a VM, but i added screenshots of the last one.
![image](https://user-images.githubusercontent.com/72886423/197507519-fce7708d-7ab9-4840-a688-60b772df2295.png)
![image](https://user-images.githubusercontent.com/72886423/197507581-6d14658a-9282-4ac8-954f-f423b11c1365.png)
![image](https://user-images.githubusercontent.com/72886423/197507687-d76f4522-fe9a-4e8a-afd8-e9a307d0f8c0.png)


[supportbundle_f3ca6d24-21d3-41ad-bb58-f99cd4accb97_2022-10-24T10-31-30Z.zip](https://github.com/harvester/harvester/files/9850929/supportbundle_f3ca6d24-21d3-41ad-bb58-f99cd4accb97_2022-10-24T10-31-30Z.zip)

Thank you!

**Comment by w13915984028** [comment]
@Eminos2 the `disk-0`, `defult/ubuntu-20.04-live-server-amd64.iso'  is not ready.

how did you add it? download from ubuntu website or upload from your local pc.

please add information of `kubectl get pods -A`  (ssh into the NODE, and `sudo -i` first) , it seems the `longhorn` CSI is not correctly working.

**Comment by w13915984028** [comment]
@Eminos2 

The below log shows that the free disk space is not enough. The vm-img (10Gib) is not ready. Not sure what has happened.

Were there many un-successful download in-between create-image ? or cancelled download.


```
logs/longhorn-system/longhorn-manager-969md/longhorn-manager.log:2022-10-24T08:22:09.402094698Z time="2022-10-24T08:22:09Z" level=debug msg="Schedule replica pvc-d2100b4c-9d1f-4e32-923f-50852bb2f4e9-r-4152a2b5 to node srv-harvester, disk e5ea5d95-4a38-45d9-bfbb-bf8b9ce7ef0a, diskPath /var/lib/harvester/defaultdisk, dataDirectoryName pvc-d2100b4c-9d1f-4e32-923f-50852bb2f4e9-42428794"

logs/longhorn-system/longhorn-manager-969md/longhorn-manager.log:2022-10-24T08:22:10.601367073Z time="2022-10-24T08:22:10Z" level=error msg="There's no available disk for replica pvc-d2100b4c-9d1f-4e32-923f-50852bb2f4e9-r-4152a2b5, size 10737418240"
```

**Comment by w13915984028** [comment]
Close first.

@Eminos2 Please feel free to reopen when you still have such issue, thanks.
